### PR TITLE
Terminal documentation

### PR DESCRIPTION
**Pull request by TheWolf534**
We finished, await your feedback and review

### CODE REVIEW

**Comment by zerothi** [comment]
Thanks for this, @TheWolf534 could you please elaborate on what you added to the `sphinx-term` source repo, what is necessary, what is nice, etc. A somewhat comprehensive overview of what you did there would be great. :)

**Comment by TheWolf534** [comment]
Hey, yeah for sure.

So, I would the necessary functionality I implemented was the run fast and restart buttons which were not available by default on the original JS code made by Ines which is then utilized by sphinx-term, thus I forked the original terminal repo which was used as a submodule in sphinx term, retrofitted a solution base on the mkdocs module termynal and changed the submodule sphinx-term is using.

Some other nice additions are:
- Adding different window styles for the termynal, eg. mac, windows, using the `:windows:` and `:unix:` tags.
- Added the ability to change the window title.
- Fixed the JS for the lineDelay function which was not working properly.

File: docs/os/Manu.rst.include [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+In this list you should not be able to see the 'NewPythonFolder' you have just deleted.
+
+7. Which Python version? (python --version or python3 --version)
+==================================
+
+Check which Python version you have.  **Note that this might change depending on what system you 
+have and wether you have more than one Python version installed**
+
+You will have to run either:
+   .. code-block::
```
Comment (zerothi) [inline docs/os/Manu.rst.include:187]: When you want more in one, I would do a comment:

```
.. code-block::
     python
     # or 
     python3
```
it is more compressed, and more readable.

**Review by zerothi — CHANGES_REQUESTED** [review]
Some comments, looking great!
Once what we talked about today is also handled we will go through how it should be placed in the site.

**Comment by TheWolf534** [comment]
@zerothi Finished making all the GIFs for PS, CMD and Bash, and incorporated them all into the page 😊

**Comment by zerothi** [comment]
Merged in 0f50b70388f181ddd78d799e94901897026e231f

**Pull request closed without merging**